src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    &:hover {
+      transition-duration: 300ms;
+      transform: scale(2);
+      color: #34568b;
+    }
```

```suggestion
    border-radius: 5px;
    margin-bottom: 48px;

    &:hover {
      transition-duration: 300ms;
      transform: scale(1.2);
      color: #34568b;
    }
```

transition-duration: transform 300ms;  - цю властивість краще задати на основний селектор, тоді transition відпрацьовуватиме в обидві сторони